### PR TITLE
Added CWS_DISABLE_RENEWAL_CHECKS env var to CWS 

### PR DESCRIPTION
**Pull request by marianunez**
#### Summary

Added `CWS_DISABLE_RENEWAL_CHECKS` env var to CWS spinwick deployments. 

```release-note
NONE
```

### CODE REVIEW

File: server/config.go [hunk 1]
```diff
@@ -32,6 +32,7 @@ type CWS struct {
 	CWSBlapiToken             string
 	CWSLicenseGeneratorURL    string
 	CWSLicenseGeneratorKey    string
+	CWSDisableRenewalChecks   string
```
Comment (nickmisasi) [inline server/config.go:35]: Indentation looks off

**Review by nickmisasi — APPROVED** [review]
Small thing but lgtm

**Review by cpanato — APPROVED** [review]
thanks!

**Comment by cpanato** [comment]
Adding @stylianosrigas because they need to be aware and eventually deploy this change

**Comment by stylianosrigas** [comment]
@cpanato thanks, will you cut a release after a merge?

**Comment by cpanato** [comment]
need to sync with @marianunez for that, don't know when she wants the change to be available

**Comment by stylianosrigas** [comment]
Awesome let me know and I will deploy ;)

**Comment by marianunez** [comment]
> need to sync with @marianunez for that, don't know when she wants the change to be available

@cpanato @stylianosrigas would like to deploy as soon as you guys have a chance. Thanks!

**Pull request merged**